README.md

```diff
@@ -0,0 +1,189 @@
+
+<div align="center">
+  <h3><b>Budget-App README</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Please deploy the application on one of the free host services and please make sure that it's working properly, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=is%20tested%20adequately.-,Deploy%20the%20project,-%2C%20and%20test%20for) requirement